deploy/demo/quickstart/aad-pod-identity-quickstart.sh

```diff
@@ -0,0 +1,150 @@
+		echo "Exiting script."
+		exit 1
+	fi
+}
+
+# function to validate Managed Serivce Identity
+validate_msi_account () {
+	# ${1//\"/} is used to remove quotes from string, which is needed for the az commands
+	echo "Validating the existence of the Azure Managed Service Identity ${1}..."
+	MSINAMEQUERY=$(az ad sp list --display-name ${1//\"/} -o json | jq -r '.[].displayName')
+	# echo "MSINAMEQUERY is ${MSINAMEQUERY}"
+	# echo ${1//\"/}
+	# exit
+	if [ $MSINAMEQUERY == ${1//\"/} ]; then
+		echo "Azure Managed Service Identity ${1} already exists. Gathering MSI details..."
+		read MSINAME MSICLIENTID MSIRESOURCEID MSIPRINCIPALID < <(echo $(az ad sp list --display-name ${1//\"/} -o json | jq -r '.[].displayName, .[].appId, .[].alternativeNames[1], .[].objectId'))
+	else
+		echo "Azure Managed Service Identity ${1} does not exist. Creating MSI now..."
+		read MSINAME MSICLIENTID MSIRESOURCEID MSIPRINCIPALID < <(echo $(az identity create -g ${2//\"/} -n ${1//\"/} -o json | jq -r '.name, .clientId, .id, .principalId'))
+		echo "Created MSI account ${MSINAME}, and allowing account to propagate the AAD directory..."
+		sleep 30
+	fi
+}
+
+echo "Validating action of the script..."
+# Determine if demo script is to deploy demo env or clean up demo env
+# ${1//\"/} is used to remove quotes from string, which is needed for the az commands
+if [ $1 = "deploy" ]; then
+	echo "Script is to deploy AAD Pod Idenity demo."
+	validate_demo_config_file "$2"
+	validate_azure_resource_group $ASSETSRESOURCEGROUPNAME $LOCATION # Creating/Validating the Assets Resource Group
+	validate_azure_resource_group $ACCESSRESOURCEGROUPNAME $LOCATION # Creating/Validating the Access Resrouce Group
+	validate_azure_resource_group $NOACCESSRESOURCEGROUPNAME $LOCATION # Creating/Validating the No Access Resource Group
+	validate_msi_account $MSINAME $ASSETSRESOURCEGROUPNAME # Creating/Validating the Managed Service Identity
+	echo "Assigning MSI ${MSINAME} Contributor role access to access resource group ${ACCESSRESOURCEGROUPNAME}..."
+	ACCESSRESOURCEGROUPID=$(az group create -l ${LOCATION//\"/} -n ${ACCESSRESOURCEGROUPNAME//\"/} --query id -o tsv)
+	ASSIGNCONTRIBUTOR=$(az role assignment create --assignee-object-id ${MSIPRINCIPALID//\"/} --scope ${ACCESSRESOURCEGROUPID//\"/} --role Contributor 2>/dev/null)
+	echo "Completed assigning MSI ${MSINAME} Contributor role access to access resource group ${ACCESSRESOURCEGROUPNAME}."
+	echo "Assigning MSI ${MSINAME} Reader role access to access resource group ${NOACCESSRESOURCEGROUPNAME}..."
+	NOACCESSRESOURCEGROUPID=$(az group create -l ${LOCATION//\"/} -n ${NOACCESSRESOURCEGROUPNAME//\"/} --query id -o tsv)
+	ASSIGNREADER=$(az role assignment create --assignee-object-id ${MSIPRINCIPALID//\"/} --scope ${NOACCESSRESOURCEGROUPID//\"/} --role Reader 2>/dev/null)
+	echo "Completed assigning MSI ${MSINAME} Contributor role access to access resource group ${ACCESSRESOURCEGROUPNAME}."
```

Comment:
Should this be `Completed assigning MSI ${MSINAME} Reader role access to access resource group ${NOACCESSRESOURCEGROUPID}`